tests/chainer_tests/training_tests/extensions_tests/test_logreport.py

```diff
@@ -0,0 +1,60 @@
+import shutil
+import tempfile
+import unittest
+
+import mock
+
+from chainer import testing
+from chainer import training
+from chainer.training import extensions
+
+
+class TestLogReport(unittest.TestCase):
+
+    def setUp(self):
+        self.temp_dir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        shutil.rmtree(self.temp_dir)
+
+    def test_trigger(self):
```

Comment:
It is certainly true that this test passes if `_write` is called with appropriate arguments. But it also passes if arguments are wrong but `_write` is never called. What I intended is to create directly call `log_report._write` with expected argument to check arguments fed to the mock object in `test_write_trigger`. You can check it with [assert_called_with](https://docs.python.org/3.6/library/unittest.mock.html#unittest.mock.Mock.assert_called_with). I am sorry my comment was insufficient.